e2e/workflowTest/edit-listing-test/attribute-value-options-by-criteria.spec.js

```diff
@@ -7,14 +7,15 @@ let criteria;
 let hooks;
 let login;
 let page;
-const inputs = require('./dataProviders/testProcedure-options-dp');
+const inputs = require('./dataProviders/attribute-value-options-dp');
 
 inputs.forEach((input) => {
   const { criteriaName } = input;
   const { testProcedureOptions } = input;
   const { id } = input;
   const { criteriaOld } = input;
   const { cures } = input;
+  const { testToolsOptions } = input;
```

Comment:
Not required, but this could be combined into a single assignment
```suggestion
  const { criteriaName ,
 testProcedureOptions ,
 id ,
 criteriaOld ,
 cures ,
testToolsOptions } = input;
```